src/AutoVizuA11y.tsx

```diff
@@ -5,18 +5,23 @@
  * Other licensing options may be available, please reach out to data-viz@feedzai.com for more information.
  */
 
-import React, { useEffect, useState } from "react";
-import ReactDOM from "react-dom/client";
+import React, { useEffect, useMemo, useRef, useState } from "react";
 
-import { addsAriaLabels, keyHandler, levelChart, levelNav } from "./components";
-import { arrayConverter, newId } from "./utils";
+import { addAriaLabels, switchToChartLevel } from "./components";
+import { arrayConverter } from "./utils";
 
 import ShortcutGuide from "./ShortcutGuide";
 import { generateDescriptions } from "./components/descriptions/DescriptionsGenerator";
 import { insightsCalculator } from "./utils/insightsCalculator";
-import { descriptionsChanger } from "./components/descriptions/Descriptions";
+import { descriptionsKeyHandler } from "./components/descriptions/DescriptionsKeyHandler";
 
-import "./style/AutoVizuA11y.css";
+import "./assets/style/AutoVizuA11y.css";
```

Comment:
Probably just a minor nitpick, but wouldn't the file "seem" more organized if the css imports were moved to the bottom of this initial import stage?

We do not have a rule for this but, you know, "modules first, styles later" kind of seem like a good approach to me.

Let me know what you think.